src/dsl/index.ts

```diff
@@ -1 +1 @@
-export * from './OpenApiBuilder';
+export * from './OpenApiBuilder.js';
```

Comment:
Do you really need the `.js` extension for any use case?

I was following the TS convention of avoiding extensions on imports to refer TS/JS files depending on the context (edit time)/(build time).

Reply:
This prevents issues with module resolution in node ESM exports. It's _very_ annoying, I agree

https://stackoverflow.com/questions/62619058/appending-js-extension-on-relative-import-statements-during-typescript-compilat

More context there, but for now this is the simplest fix.

Reply:
Thanks for the explanation. Agreed, let's keep the extensions.